src/devices/SocketCan/SocketCan.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(DefaultBindingTfms)</TargetFrameworks>
+    <TargetFrameworks>net5.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Do you really need this change (this removes netstandard support for this binding)? If so, you have to change the dependent tests and/or samples as well.

Reply:
Yes. As discussed in #1277, this would be required.

At first I wanted to get the implementation "right" (for my requirements). I will take a look at the changes proposed by @krwq (see [comment](https://github.com/dotnet/iot/pull/1277#issuecomment-731089781)) next.

But since I had a hard time finding any .NET (Core/5) library for SocketCan and the project here does not use any of the GPIO parts (but is part of [Iot.Device.Bindings](https://www.nuget.org/packages/Iot.Device.Bindings/) "This package provides a set of Device Bindings that use System.Device.Gpio package to communicate with a microcontroller."), I'm not totally convinced that this code should stay here. Does a better place exist?